create_dataset.py

```diff
@@ -32,10 +55,10 @@ def read_context():
 
     # move _stdout.txt log file to dataset
     shutil.copyfile('_stdout.txt', os.path.join(dataset_name, '_stdout.txt'))
-
     # generate .dataset.json data
     dataset_json_data = generate_dataset_json_data(dataset_json_files, VERSION)
     dataset_json_data['label'] = dataset_name
+    dataset_json_data['location'] = {'type': 'Polygon', 'coordinates': [coordinates]}
     print(json.dumps(dataset_json_data, indent=2))
```

Comment:
Changes to overwrite the location polygon in dataset.json to accurately reflect what was used in `stackSentinel.py` (previously was taken from SLC scene)